src/port1.0/portconfigure.tcl

```diff
@@ -429,10 +429,20 @@ proc portconfigure::configure_get_universal_archflags {} {
     global configure.universal_archs
     set flags ""
     foreach arch ${configure.universal_archs} {
+        set option ""
+        # passthrough custom options
+        if {[string match "-*" $arch]} {
```

Comment:
I'm a little surprised at the flag passthrough in `configure.universal_archs`. Isn't this what `configure.cflags`/`configure.cppflags`/`configure.cxxflags`/`configure.ldflags` are for? What's your use-case for those?